### PR TITLE
bugfix/15074-lastPrice-color

### LINKED ISSUE #15074: lastPrice color property is not applied properly
#### Expected behaviour
Always apply the color for the `lastPrice` line as declared in options.

#### Actual behaviour
After two clicks color changes.

#### Live demo with steps to reproduce
Click twice the `currentPriceIndicator` button.
http://jsfiddle.net/BlackLabel/n1wz7hbe/

#### Product version
Highstock JS v9.0.0 (2021-02-02)

#### Affected browser(s)
All

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed #15074, `lastPrice` color was not applied properly.

### CODE REVIEW

**Review by pawellysy — APPROVED** [review]
Looks Good!

File: ts/Stock/StockToolsBindings.ts [hunk 1]
```diff
@@ -2089,7 +2089,7 @@ var stockToolsBindings: Record<string, Highcharts.NavigationBindingsOptionsObjec
                 // line
                 lastPrice: {
                     enabled: !lastPrice,
-                    color: 'red'
+                    color: (options.lastPrice && options.lastPrice.color) || '#ff0000'
```
Comment (pawelfus) [inline ts/Stock/StockToolsBindings.ts:2092]: Is this necessary to define `color` here? It should be enough to toggle the `enabled` option from the button perspective, right?

**Review by pawelfus — CHANGES_REQUESTED** [review]

**Review by raf18seb — APPROVED** [review]
Look good 👍

**Review by pawelfus — APPROVED** [review]
Sweet! :)

**Pull request merged**